mango-core/src/main/scala/org/bdgenomics/mango/serialization/MangoKryoRegistrator.scala

```diff
@@ -25,6 +25,7 @@ import org.bdgenomics.adam.serialization._
 import org.bdgenomics.adam.util.{ TwoBitFile, TwoBitFileSerializer }
 import org.bdgenomics.formats.avro._
 import org.bdgenomics.mango.layout.CalculatedAlignmentRecord
+import org.bdgenomics.mango.tiling.ReferenceTile
 
 import scala.collection.mutable.{ ArrayBuffer, ListBuffer }
 
```

Comment:
surprised you didn't run into kryo issues with adam upgrades. I changed this quite a bit (will merge it in with variant stuff), copying from adam's new kryoregistrator.